src/Layout/Footer/styles.module.css

```diff
@@ -0,0 +1,57 @@
+.footer {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    min-height: 400px;
+    width: 100%;
+    background-color: #FDFDFD;
+}
+.footer p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
```

Comment:
You are using `font-size: 16px;` in multiple places, introduce a new `css` variable called `--regular-font-size`, same goes for `font-size: 40px`, it can be `--big-font-size`. Same goes for any value that you are using multiple times. 

Why is that important? 
Because when the designer will say "please change all the fonts to be bigger by 1px` you are doing it only in one place, and if the designer will say `let's change all the app colors from green #0F0 to red #F00", you are doing it only in one place.